settings.py

```diff
@@ -782,5 +782,5 @@ def _theme_completions(view):
                 "{}  \ttheme".format(theme),
                 theme
             )
-            completions.add(item)
+            completions.append(item)
```

Comment:
Just found the `if item not in completions` has gone (or was there). I was not yet aware of sets being able to automatically make "lists" unique. Cool feature - again learned something. 😄 

I am on trying to find a common way to create these items no matter of source without too many type conversions and list recreations.

Quotation issue seems to apply to list elements primarily.